src/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ add_subdirectory(lib-python)
 add_subdirectory(marine)
 add_subdirectory(conventional)
 add_subdirectory(ncep)
+add_subdirectory(owp_snow_obs)
```

Comment:
I think it would make sense to move this to the source directory for the "land observations" converters. Youlong has created this in the "src/land" directory for a number of land related converters he has developed. Do you agree?

Thanks!

Reply:
Agreed